zerver/lib/markdown/api_return_values_table_generator.py

```diff
@@ -45,7 +45,9 @@ def run(self, lines: List[str]) -> List[str]:
                 text: List[str] = []
                 if endpoint != "/events":
                     text = self.render_table(return_values, 0)
-                else:
+                # delete-queue is not supposed to have return
+                # values.
+                elif doc_name != "/events:delete":
                     return_values = copy.deepcopy(return_values)
```

Comment:
I don't understand this logic; shouldn't it just be `if doc_name != "/events:get" / else` , so that every endpoint but `events:get` has the default logic?

(I think also a cleaner way to write it would be to have `if doc_name == "/events:get": / else` block.)